client/cli/Cargo.toml

```diff
@@ -37,7 +37,7 @@ sp-keystore = { version = "0.10.0-dev", path = "../../primitives/keystore" }
 sc-service = { version = "0.10.0-dev", default-features = false, path = "../service" }
 sc-telemetry = { version = "4.0.0-dev", path = "../telemetry" }
 sp-keyring = { version = "4.0.0-dev", path = "../../primitives/keyring" }
-names = "0.11.0"
+names = { default-features = false, git = "https://github.com/fnichol/names", rev= "8ff6d9a0b5e52fd2454fdeac21c57f372d2da4e3" }
```

Comment:
This is going to break `unleash` though. It's fine since this PR is against `polkadot-v0.9.10`, but a fix to `master` should IMHO be to simply get rid of the crate and write the code manually (or fork it).

Reply:
The back port can be tracked in #9752